### ISSUE TITLE
Rule proposal: require closing bracket on new line

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#14]
A rule that enforces that the closing bracket of a component must be on its own line might be nice.

Good:

```jsx
<MyComponent
  firstProp="value"
  secondProp="value"
/>
```

Bad:

```jsx
<MyComponent
  firstProp="value"
  secondProp="value" />
```

**Comment by vvo** [comment]
Nice!
